packages/api/src/routers/api/logViews.ts

```diff
@@ -18,11 +19,16 @@ router.post('/', async (req, res, next) => {
     }
     const logView = await new LogView({
       name,
+      tags,
       query: `${query}`,
       team: teamId,
       creator: userId,
     }).save();
-
+    if (tags?.length) {
+      redisClient.del(`tags:${teamId}`).catch(e => {
```

Comment:
can also move this to mongoose post hook on Dashboard and LogView 🤔

Reply:
my 2c is that we should avoid caching complexity until it's warranted. I think for now avoiding caching and just reading directly from mongo would be perfect. Usually just long CH-operations warrant caching (and even then I try to push for cache-less approaches)

Though @wrn14897 maintains the backend so I'll defer to his final judgement :)

Reply:
makes sense! removed caching and used aggregation to get all tags for a team

Reply:
agreed that we should remove the caching for now unless it has a dramatic performance impact